### PR TITLE
Add first version of a wishlist app

### LINKED ISSUE #387: Wishlist
We often talk about things that would be nice to have, but that we don't spend much sponsor money on. It would be nice if each team on the website could write a list of items that they need and companies/individuals can go to our website and see what equipment we need before they throw it out and donate it to us instead.

Examples of these items that we almost always need are older laptops, sofas, outdoor furniture for the bar, power cables.

### PR DESCRIPTION
**Pull request by tykling**
Create wishes in the admin for now. Fix a few unrelated things while here:

* Update our date and time formats to be more internationally unambigous, and always show timezone.
* Also fix an old bug which meant that bornhack.dk/program redirect never worked.

Fixes #387

### CODE REVIEW

File: src/bornhack/urls.py [hunk 1]
```diff
@@ -77,6 +77,12 @@
         kwargs={"page": "village_list"},
         name="village_list_redirect",
     ),
+    path(
+        "wishlist/",
+        CampRedirectView.as_view(),
```
Comment (valberg) [inline src/bornhack/urls.py]: Should be `WishListView` right?

File: src/bornhack/settings.py [hunk 2]
```diff
@@ -69,10 +70,10 @@ def local_dir(entry):
 # USE_I18N = True
 # USE_L10N = True
 USE_TZ = True
-SHORT_DATE_FORMAT = "d/m-Y"
-DATE_FORMAT = "d/m-Y"
-DATETIME_FORMAT = "d/m-Y H:i"
-TIME_FORMAT = "H:i"
+SHORT_DATE_FORMAT = "Ymd"
+DATE_FORMAT = "l, M jS, Y"
+DATETIME_FORMAT = "l, M jS, Y, H:i (e)"
```
Comment (valberg) [inline src/bornhack/settings.py]: What is `e`?

File: src/wishlist/models.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    def camp(self):
+        return self.team.camp
+
+    camp_filter = "team__camp"
+
+    def __str__(self):
+        return self.name
+
+    def save(self, **kwargs):
+        print("inside save()")
```
Comment (valberg) [inline src/wishlist/models.py]: Print alert!

File: src/bornhack/settings.py (hunk 2 above)
Comment (tykling) [reply src/bornhack/settings.py]: timezone abbreviation, CET right now, CEST during bornhack

File: src/wishlist/models.py (hunk 3 above)
Comment (tykling) [reply src/wishlist/models.py]: welp where did that come from

**Comment by valberg** [comment]
Except for the print call - LGTM :+1:

**Comment by tykling** [comment]
great thanks! :)

**Pull request merged**